app/javascript/shared/components/tags.jsx

```diff
@@ -358,6 +380,7 @@ class Tags extends Component {
         }
         // updates searchResults array according to what is being typed by user
         // allows user to choose a tag when they've typed the partial or whole word
+        console.log('return search results')
```

Comment:
I agree with @Lukyhenson, this doesn't seem to be necessary? Why would we leave this in production?